### ISSUE TITLE
Problems reading digit '1' from images

### ISSUE THREAD

**Issue by OjasMadaan** [JaidedAI/EasyOCR#747]
Hi all, I'm currently trying to read characters from number plates - it usually works accurately. Still, for some images, it mistakenly reads the digit '1' as the letter 'L' and also makes minor mistakes elsewhere. Any suggestions to improve the accuracy? 
Basically I have a cropped image which only contains the license plate and the following code is run :
`reader = easyocr.Reader(['en'], gpu=False)
    result = reader.readtext(cropped_array)`
and the outputs look like :
<img width="410" alt="Screen Shot 2022-06-06 at 11 46 18 PM" src="https://user-images.githubusercontent.com/79814331/172154655-197aec12-b30d-4ec3-9771-61f0eb2afb2b.png">
<img width="292" alt="Screen Shot 2022-06-06 at 11 47 32 PM" src="https://user-images.githubusercontent.com/79814331/172154794-c2229005-1830-4ef9-8430-392762dc4e6e.png">

**Comment by iblub1** [comment]
You could try to:
- finetune the model on your dataset
- use some heuristics if you have expert knowledge
- change some hyperparameters for the default recognizer https://github.com/JaidedAI/EasyOCR/blob/627503d0e8aa123ad294e46502b51ff18cd11bb3/easyocr/easyocr.py#L299
